src/main/java/com/icthh/xm/tmf/ms/resource/client/OAuth2UserClientFeignConfiguration.java

```diff
@@ -1,9 +1,10 @@
 package com.icthh.xm.tmf.ms.resource.client;
 
 import feign.RequestInterceptor;
-import java.io.IOException;
 import org.springframework.context.annotation.Bean;
 
+import java.io.IOException;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version